src/index.html

```diff
@@ -9,6 +9,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
May I ask you ? Is this reference design for 'a' tags with multiple attributes? Thanks!

Reply:
Yes and not only for 'a' tags
![image](https://user-images.githubusercontent.com/80700247/222897256-1d3fca66-a63d-44c0-a880-c752a9eb8e7e.png)

Reply:
Sorry, and i've second question. Should I used **case 1** always? Or is it possible to use the **case 2** ?
![image](https://user-images.githubusercontent.com/62157487/222899168-c2848243-db9e-4f1b-ab57-052406fd2250.png)